cset_openalex_augmentation_dag.py

```diff
@@ -34,6 +36,29 @@
 args["retries"] = 1
 
 
+def get_updated_version() -> str:
+    """
+    Get the latest semver string for the delivery, and update the corresponding airflow Variable. By default,
+    we bump the minor version with each delivery, but this function allows us to manually set a different
+    part of the semver string to bump in the variable
+    :return: Updated semver string
+    """
+    oa_variable = "cset_openalex_version"
+    version_config = Variable.get(oa_variable, deserialize_json=True)
+    version_to_updater = {
+        "major": semver.bump_major,
+        "minor": semver.bump_minor,
+        "patch": semver.bump_patch,
+    }
```

Comment:
Since this is constant across runs, I think this should be pulled out of the function and that its purpose be made clearer (to map update types to updater functions).

Reply:
What would you suggest?